test/AuthTokenVerifier.test.ts

```diff
@@ -45,56 +43,53 @@ describe("Testing AuthTokenVerifier", () => {
       sub: secondDID
     }
 
-    firstSigner = new JWT(firstKeys);
-    secondSigner = new JWT(secondKeys)
-    firstClaimToken = await firstSigner.sign(firstPayload, { algorithm: 'ES256' });
-    secondClaimToken = await secondSigner.sign(secondPayload, { algorithm: 'ES256' });
-  })
-
-  it("Checks if basic signing verification works", async () => {
-    const testSignature = await secondSigner.sign('Initial signed content', { algorithm: 'ES256' })
-    const secondPubKey = secondKeys.publicKey
-    const decoded = await secondSigner.verify(testSignature, secondPubKey)
-    
-    assert.strictEqual(decoded, 'Initial signed content')
+    firstClaim = await createSignWithEthersSigner(firstWallet)(firstPayload);
+    secondClaim = await createSignWithEthersSigner(secondWallet)(secondPayload);
```

Comment:
I think that it is good that we are testing Ethers signature now but I think we should also test with a JWT signer which is what I think we would get with `new JWT(keys);`. Basically, we need to test two signature approaches?